cms/templates/manage_users_lib.html

```diff
@@ -0,0 +1,181 @@
+                    <span class="msg-you">${_("You!")}</span>
+                % endif
+              </span>
+            </span>
+          </span>
+
+          <div class="item-metadata">
+            <h3 class="user-name">
+              <span class="user-username">${user.username}</span>
+              <span class="user-email">
+                <a class="action action-email" href="mailto:${user.email}" title="${_("send an email message to {email}").format(email=user.email)}">${user.email}</a>
+              </span>
+            </h3>
+          </div>
+
+          % if allow_actions:
+          <ul class="item-actions user-actions">
+            % if is_instructor:
+              % if len(instructors) > 1:
+                <li class="action action-role">
+                  <a href="#" class="make-staff admin-role remove-admin-role">${_("Remove Admin Access")}</span></a>
+                </li>
+              % else:
+                <li class="action action-role">
+                  <span class="admin-role notoggleforyou">${_("Promote another member to Admin to remove admin rights")}</span>
+                </li>
+              % endif
+            % elif is_staff:
+              <li class="action action-role">
+                <a href="#" class="make-instructor admin-role add-admin-role">${_("Add Admin Access")}</span></a>
+                <a href="#" class="make-user admin-role remove-admin-role">${_("Remove Staff Access")}</span></a>
+              </li>
+            % else:
+              <li class="action action-role">
+                <a href="#" class="make-staff admin-role add-admin-role">${_("Add Staff Access")}</span></a>
+              </li>
+            % endif
+            <li class="action action-delete ${"is-disabled" if request.user.id == user.id and is_instructor and len(instructors) == 1 else ""}">
+                <a href="#" class="delete remove-user action-icon" data-tooltip="${_("Remove this user")}"><i class="icon-trash"></i><span class="sr">${_("Delete the user, {username}").format(username=user.username)}</span></a>
+            </li>
+          </ul>
+          % elif request.user.id == user.id:
+          <ul class="item-actions user-actions">
+            <li class="action action-delete">
+                <a href="#" class="delete remove-user action-icon" data-tooltip="${_("Remove me")}"><i class="icon-trash"></i><span class="sr">${_("Remove me from this library")}</span></a>
+            </li>
+          </ul>
+          % endif
+
+        </li>
+        % endfor
+      </ol>
+
+      % if allow_actions and len(all_users) == 1:
+      <div class="notice notice-incontext notice-create has-actions">
+        <div class="msg">
+          <h3 class="title">${_('Add More Users to This Library')}</h3>
+          <div class="copy">
+            <p>${_('Adding team members makes content authoring collaborative. Users must be signed up for Studio and have an active account. ')}</p>
```

Comment:
@catong I'm assuming this should now be `_('Grant other members of your course team access to this library. New library users must have an active {studio_name} account.').format(studio_name=settings.STUDIO_SHORT_NAME)` since it appears all references to "Studio" are being made cutomizable (OPEN-423 / PR 6309).